ci_config.json

```diff
@@ -869,5 +869,8 @@
       "wayland:tests=false",
       "wayland:dtd_validation=false"
     ]
+  },
+  "x-plane-sdk": {
+    "_comment": "- nothing special."
```

Comment:
If there's no config being passed to the CI then I'd expect to not have an entry at all.

Reply:
Ah okay, wasn't sure of the procedure. I'll chop that out.